src/nih_wayfinding/app/scripts/geocoder/search-service.js

```diff
@@ -16,8 +16,23 @@
         ].join(',');
 
         // Public Interface
+        // TODO: Expose the bbox, outFields and categories vars as config options
```

Comment:
I'm not sure what you mean here? There is a bbox private var that is used to create both the search/suggest urls.